src/test/java/org/assertj/core/internal/strings/Strings_assertContainsNotOnlyWhitespaces_Test.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2017 the original author or authors.
+ */
+package org.assertj.core.internal.strings;
+
+import com.tngtech.java.junit.dataprovider.DataProvider;
+import com.tngtech.java.junit.dataprovider.DataProviderRunner;
+import org.assertj.core.internal.StringsBaseTest;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.assertj.core.error.ShouldNotBeWhitespaces.shouldNotBeWhitespaces;
+import static org.assertj.core.test.TestData.someInfo;
+
+@RunWith(DataProviderRunner.class)
+public class Strings_assertContainsNotOnlyWhitespaces_Test extends StringsBaseTest {
```

Comment:
rename to `Strings_assertDoesNotContainOnlyWhitespaces_Test`

Reply:
Done